Resources/Locale/en-US/deltav/store/uplink-catalog.ftl

```diff
@@ -1,6 +1,9 @@
 # Utility
 uplink-reinforcement-radio-nukie-mouse-name = Mouse Operative Reinforcement Teleporter
 uplink-reinforcement-radio-nukie-mouse-desc = Calls in a specially trained mouse operative to assist you.
+
+uplink-reinforcement-radio-syndicate-paramedic-name = Syndicate Paramedic Reinforcement
+uplink-reinforcement-radio-syndicate-paramedic-desc = Calls in a questionably trainled syndicate paramedic to assist you.
```

Comment:
I feel like the description here can be improved. This is a good opportunity to add in loreful flavor text, rather than the usual jokes. Aside from the Typo, perhaps it can be rewritten to have more themes in common with the cybersun paramed outfit and its flavor texts.